content/docs/latest/keyless/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Keyless Integration with Platforms and Applications
+short: Keyless Authentication
+kind: keyless
+---
+
```

Comment:
For now, it would be great to have a couple of lines explaining to a reader the broad types of problems this content can help them solve.

Reply:
I added a summary - please review.

Reply:
Reading this section more carefully, I'm getting tripped up a little on "Keyless Authentication" (which is a great marketing term, but not literally accurate). Technically one could argue that SVIDs are in fact keys :)

Consider:
```
title: Using SVIDs To Authenticate to Other Platforms and Applications
short: SVID Authentication
kind: keyless
---
The SVIDs generated by SPIRE can be used by a workload to authenticate to some other third party platforms and applications, avoiding the need to generate, distribute and rotate additional credentials for that application. These tutorials show how this can be achieved. 
```